codex-rs/core/src/codex.rs

```diff
@@ -601,8 +601,8 @@ impl AgentTask {
             self.handle.abort();
             let event = Event {
                 id: self.sub_id,
-                msg: EventMsg::Error(ErrorEvent {
-                    message: "Turn interrupted".to_string(),
+                msg: EventMsg::BackgroundEvent(BackgroundEventEvent {
```

Comment:
Let's introduce a new variant in `EventMsg` for this and then let the UI decide how it wants to present it.